vdb/lib/cve.py

```diff
@@ -296,13 +298,78 @@ def severity_to_impact(severity: str) -> CiaType:
     return CiaType.LOW
 
 
+def severity_to_impact4(severity: str) -> str:
+    """Method to convert severity string to a valid impact enum string"""
+    if not severity:
+        return "NONE"
+    if severity.lower() in ("critical", "high", "medium", "moderate"):
+        return "HIGH"
+    return "LOW"
+
+
+def fix_maturity(maturity: str) -> str:
+    if not maturity:
+        return "NOT_DEFINED"
+    if maturity == "POC":
+        return "PROOF_OF_CONCEPT"
+    return maturity.upper()
+
+
+def fix_attack_vector(attack_vector: str) -> str:
+    if attack_vector == "ADJACENT_NETWORK":
+        return "NETWORK"
+    return attack_vector.upper()
+
+
 def to_metrics(avuln: Vulnerability) -> Metrics:
     metrics_list = []
-    if avuln.cvss_v3.vector_string and avuln.cvss_v3.vector_string.startswith(
+    if avuln.cvss4_vector_string:
```

Comment:
@prabhu This is going to be a problem for CSAF as CSAF 2.0 only allows cvss v3.0 or v3.1.

Reply:
Interesting. Are both the values not getting included?